tests/rptest/tests/random_node_operations_test.py

```diff
@@ -412,20 +396,22 @@ def enable_write_caching_testing():
 
         # create some initial topics
         self._create_topics(10)
-        regular_topic = TopicSpec(name='tp-workload-deletion',
-                                  partition_count=self.max_partitions,
-                                  replication_factor=3,
-                                  cleanup_policy=TopicSpec.CLEANUP_DELETE,
-                                  segment_bytes=default_segment_size,
-                                  redpanda_remote_read=with_tiered_storage,
-                                  redpanda_remote_write=with_tiered_storage)
+        regular_topic = TopicSpec(
+            name='tp-workload-deletion',
+            partition_count=self.max_partitions,
+            replication_factor=3,
+            cleanup_policy=TopicSpec.CLEANUP_DELETE,
+            segment_bytes=default_segment_size,
+            # disable remote reads/writes in this topic as
+            # the fast partition movement one has them enabled
```

Comment:
Could you confirm my understanding please?
1. We remove the without-tiered-storage variant of the test to save on test execution time
2. We still would like to test with and without it, so we keep one of the topics in the test without tiered storage

Reply:
it is exactly like that